src/couch_log/test/couch_log_test_util.erl

```diff
@@ -12,6 +12,7 @@
 
 -module(couch_log_test_util).
 -compile(export_all).
+-compile(nowarn_export_all).
```

Comment:
Please replace `-compile(export_all)` with:
```
-export([
   start/0,
   stop/1,
   with_level/2,
   with_config_listener/1,
   wait_for_config/0,
   last_log_key/0
]).
```

Reply:
Actually, both `last_log/0` and `with_meck/0` also need to be exported, so the full export attribute becomes:
```
-export([
    start/0,
    stop/1,
    last_log/0,
    last_log_key/0,
    wait_for_config/0,
    with_config_listener/1,
    with_level/2,
    with_meck/2
]).
```